rest-tests/src/test/resources/features/common.feature

```diff
@@ -32,3 +32,18 @@ Feature: Common scenarios
     Then creates SF lead with first name: "John", last name: "Doe", email: "jdoe@acme.com" and company: "ACME"
     Then validates SF to DB created new lead with first name: "John", last name: "Doe", email: "jdoe@acme.com"
 	Then cleans after SF to DB, removes user with first name: "John" and last name: "Doe"
+
+  @integrations-s3-to-s3
+  Scenario: S3 - S3 integration
+    Given cleans S3 to S3 scenario, removes two sample buckets with names: "syndesis-bucket-in, syndesis-bucket-out"
```

Comment:
- It doesn't seem like 3rd person, could we omit the `s` from all the verbs? :)

- And pls put it to separate feature file named like `integration-s3-to-s3`.